include/dr_param/yaml.hpp

```diff
@@ -114,6 +114,7 @@ inline YamlResult<void> mergeYamlNodes(YAML::Node && map_a, YAML::Node map_b) {
 	return mergeYamlNodes(map_a, map_b);
 }
 
+}
```

Comment:
Is this breaking anything?

Reply:
This looks fine since the other test cases are not effected.

Reply:
Hmm, I'm confused. How can adding an unmatched closing brace not break the header?

Reply:
sorry. I was checking the wrong file. I have updated it now.